Split outputs

ngam opened the pull request:
Fixes #5 

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

Commit:
split outputs

Commit:
MNT: Re-rendered with conda-build 3.21.7, conda-smithy 3.16.1, and conda-forge-pinning 2022.01.18.20.31.32

Commit:
bash bash

Commit:
edits

Commit:
try env var

Commit:
MNT: Re-rendered with conda-build 3.21.7, conda-smithy 3.16.1, and conda-forge-pinning 2022.01.18.20.31.32

Commit:
edits

Commit:
edits

Commit:
ready to push

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

ngam commented:
@dopplershift @ocefpaf (mainly because you both helped me a lot) or anyone in @conda-forge/core, could you please quickly double check on splitting this recipe to address issue #5? Thanks!

ngam commented:
@conda-forge-admin, please rerender

ngam commented:
I tested it locally and everything is fine. However, the tests don't run for coreutils if I split it this way for some reason... and I am not sure how to fix that.

asmeurer commented on recipe/meta.yaml:
```diff
@@ -14,16 +14,36 @@ build:
+      commands:
+        - gln --relative --help
+        - gwc --version
+        - gcat --version
```
Can we add a test that `ln` is not coming from this package?

ngam replied:
Sure, fwiw I tested exactly that locally 😄

ngam replied:
```

GNU coreutils online help: <https://www.gnu.org/software/coreutils/>
Full documentation <https://www.gnu.org/software/coreutils/ln>
or available locally via: info '(coreutils) ln invocation'
ln (GNU coreutils) 8.22
Copyright (C) 2013 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Mike Parker and David MacKenzie.
ln (GNU coreutils) 9.0
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

```

asmeurer commented:
> I tested it locally and everything is fine. However, the tests don't run for coreutils if I split it this way for some reason... and I am not sure how to fix that.

Possibly a bug in conda build?

isuruf committed:
rename top level to coreutils-split

ngam committed:
add ln vs gln version only on linux

ngam commented:
@isuruf 


Oh never mind! Thanks a lot

ngam commented:
> > I tested it locally and everything is fine. However, the tests don't run for coreutils if I split it this way for some reason... and I am not sure how to fix that.
> 
> Possibly a bug in conda build?

fixed in 5785ff9 by isuruf

ngam commented:
@asmeurer could you check if everything is working correctly on your end when you get a chance? We now have the two outputs and soon we will have them in all arches (arm, ppc, aarch) except windows. Let me know if anything could be fixed --- also, happy to add you as maintainer here if you wish

asmeurer commented:
Seems to work for me (Intel Mac).
